docs/advanced-guides/cloud-in-a-box.md

```diff
@@ -45,8 +45,12 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+The images currently download and install the 
+[latest release of the installation-scripts](https://github.com/osism/cloud-in-a-box),
+therefore it is mandatory to update the installation media at least when the underlying Ubuntu operating 
+system release changes.  The installation of historical releases is currently not supported. 
```

Comment:
I think we should mention here that this is not planed for the next at least 12 months. I think we will have Ubuntu 24.04 not before the Q3 of 2024.

Reply:
Yes